login/pkg/server.go

```diff
@@ -40,6 +46,8 @@ func (s *Server) Run() error {
 			{
 				apiV0User.POST("register", s.CreateAPI.Handler)
 				apiV0User.POST("login", s.LoginAPI.Handler)
+				apiV0User.POST("forgot_password", s.ForgotPasswordAPI.Handler)
+				apiV0User.Use(s.AuthResetPasswordAPI.Handler()).PATCH("update_password", s.UpdatePasswordAPI.Handler)
```

Comment:
It's possible to swap to `PATCH` first and then `.Use()` middleware? I think it will be more readable for programmers to see the route `path` first

Reply:
No. If u put `.Use() `after the middleware it would be called after the handler.